Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:20.04
 
-ARG TARGETARCH=amd64
+ARG TARGETARCH
```

Comment:
Default value is removed because this is a special argument which is populated by buildx plugin when --platform argument is used. If it has a default value assigned, it's not overwritten properly.